### PR TITLE
changed the file to read from a R script

### PR DESCRIPTION
**Pull request by sophieball**
I also changed R's way of sending input data
#15

### CODE REVIEW

File: src/receive_data.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Lint as: python3
+"""
+Receive data from R script
+"""
+import io
+import pandas as pd
+import sys
+
+def receive_data():
+  print("Reading data.")
```
Comment (CaptainEmerson) [inline src/receive_data.py:10]: turn this to a log statement, as you have elsewhere?

File: src/receive_data.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Lint as: python3
+"""
+Receive data from R script
```
Comment (CaptainEmerson) [inline src/receive_data.py:3]: "Receive data from standard input in CSV format". Doesn't really matter where it's coming from.

File: src/receive_data.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+"""
+import io
+import pandas as pd
+import sys
+
+def receive_data():
```
Comment (CaptainEmerson) [inline src/receive_data.py:9]: how about adding a test for this?

File: src/BUILD [hunk 4]
```diff
@@ -2,6 +2,12 @@ load("@rules_python//python:defs.bzl", "py_binary")
 load("@rules_python//python:defs.bzl", "py_test")
 load("@deps//:requirements.bzl", "requirement")
 
+py_library(
```
Comment (CaptainEmerson) [inline src/BUILD:5]: train_classifier needs a dependency on this

Comment (sophieball) [reply src/BUILD:5]: I added it in main/BUILD

File: src/receive_data.py (hunk 1 above)
Comment (sophieball) [reply src/receive_data.py]: Done

File: src/receive_data.py (hunk 3 above)
Comment (sophieball) [reply src/receive_data.py:10]: Added.

File: src/receive_data.py (hunk 2 above)
Comment (sophieball) [reply src/receive_data.py]: Changed.

File: src/BUILD [hunk 5]
```diff
@@ -154,3 +154,13 @@ py_library(
     ],
     visibility = ["//visibility:public"]
 )
+
+py_test(
+    name = "test_receive_data",
```
Comment (CaptainEmerson) [inline src/BUILD:159]: Is there a test suite that pulls together all individual tests like this one? If so, please hook this one in. If not, there should be. The CI should run that mega-suite.

Comment (sophieball) [reply src/BUILD:159]: I'll open another PR for tests

**Pull request merged**